E-gzamin-frontend/src/components/Home.tsx

```diff
@@ -16,14 +16,27 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: theme.palette.background.default,
   },
   mainHeaders:{
-    color: theme.palette.text.primary,
-    marginLeft: "3rem",
+    color: theme.palette.text.primary
   },
   testIcon: {
     fill: theme.palette.text.primary,
   },
+
 }));
 
+type HeaderType = {
+  content: string;
+}
+
+function Header(props: HeaderType){
+const {content} = props;
+const styles = useStyles();
+
+return <Typography className={styles.mainHeaders} variant="h3" component="h2" style={{margin:"1vh 3vw"}}>
```

Comment:
- Header wydaje się reużywalnym komponentem, dlatego można go chyba przenieść do oddzielnego pliku.
- dodałeś style={{margin:"1vh 3vw"}} co powoduje że mieszamy dwa rodzaje dodawania styli. Fajnie byłoby trzymać się tego useStyles.
- jeszcze jedna sprawa z variantem i componentem ale to pogadamy na priv.